admittance_controller/src/admittance_controller.cpp

```diff
@@ -193,6 +224,8 @@ CallbackReturn AdmittanceController::on_configure(
     return CallbackReturn::ERROR;
   }
 
+  admittance_->convert_damping_ratio_to_damping();
```

Comment:
Please add a comment:

// Convert the optional damping ratio (if given) to mass/spring/damper representation